docs/content/reference/versioning-support.md

```diff
@@ -11,18 +11,18 @@ There are different components that might require independent versioning and sup
 
 ### HO
 
-- A HO is intended to be released for each OCP release.
-- A HO released for OCP N minor release must support N, N-1 and N-2 OCP minor releases.
-- A HO release if anything, will only do best effort to support future OCP versions.
+- A HO is intended to be released for each OCP minor release.
+- A HO released for OCP N minor release must support guest clusters of N, N-1 and N-2 OCP minor releases.
+- A HO must be updated before future OCP minor release can be deployed or updated. For example, a HO released for OCP N minor release does not support future guest cluster N+1 minor releases. A HO must be updated to match the N+1 release before guest clusters can be deployed or updated to N+1.
 
 ### CPO
 
 - A CPO is released as part of each [OCP payload release](https://amd64.ocp.releases.ci.openshift.org/).
 
 ### CLI
 
-- A CLI is intended to be released as part of any HO release.
-- The CLI is a helper utility for dev purposes. No compatibility policies are guaranteed.
+- A CLI is intended to be released as part of every HO release.
+- A CLI is only guaranteed to be compatible with the HO release it is tied to. For example, CLI compatiblity with N-1 and N+1 HO minor releases are not guaranteed.
```

Comment:
Again here, "HO release" isn't really defined

Reply:
i understand what you mean. I'm trying to communicate that the hypershift cli needs to match the version of the HO that is deployed.

In the case of self managed, this is the hypershift cli released by MCE that matches the HO also shipped by MCE. I don't know what the equivalent release is for managed, which is why the generic term "HO release" is used.